sources/packages/web/src/assets/css/base.scss

```diff
@@ -219,3 +219,18 @@ ul.no-bullets {
   max-height: none !important;
   max-width: none !important;
 }
+
+.v-banner-success {
+  background-color: $banner-background-success !important;
+  border-width: 3px !important;
+  border-color: $banner-color-success !important;
+  border-radius: 8px !important;
+}
+
+.v-button-success {
```

Comment:
v-btn has a `color="success"`, is there a reason to not use it?

Reply:
we can create theme colors as per figma to use. 
![image](https://user-images.githubusercontent.com/54600590/168916554-aa665246-cb0b-4be5-b492-f513067cfed0.png)